app/routes.php

```diff
@@ -406,7 +406,8 @@ function( Application $app, Request $request ) use ( $ffFactory ) {
 			$request->get( 'zahlweise', '' ),
 			intval( $request->get( 'periode', 0 ) ),
 			$validationResult->isSuccessful(),
-			$trackingInfo
+			$trackingInfo,
+			$request->get( 'addressType', DonorName::PERSON_PRIVATE )
```

Comment:
Not sure it's appropriate to use this constant here. I'd need to double check which I can't do ATM, but in general domain stuff should not be used outside the domain and the UCs around it, ie in routes or presenters. Changing the value of the constant should not require matching changes in the js (or in the databse).

There are a few other such cases scattered around... not a huge issue, yet still subtle corruptions of the architecture